networkit/graph.pyx

```diff
@@ -930,6 +1008,50 @@ cdef class Graph:
 			raise Exception("Attribute name has to be a string")
 		self._this.detachNodeAttribute(stdstring(name))
 
+def GraphFromCoo(inputData, n=0, bool_t weighted=False, bool_t directed=False, bool_t edgesIndexed=False):
+	"""
+	graphFromInputData(inputData, n):
```

Comment:
function name and doc string are inconsistent (and also missing some parameters)